pkg/vere/io/ames.c

```diff
@@ -3126,6 +3200,11 @@ _ames_io_exit(u3_auto* car_u)
   uv_close(&sam_u->had_u, _ames_exit_cb);
   uv_close((uv_handle_t*)&sam_u->sun_u.dns_u, 0);
   uv_close((uv_handle_t*)&sam_u->sun_u.tim_u, 0);
+  uv_close((uv_handle_t*)&sam_u->nat_u.tim_u, 0);
+
+  if (uv_is_active((uv_handle_t*)&sam_u->nat_u.pol_u)) {
+    uv_close((uv_handle_t*)&sam_u->nat_u.pol_u, 0);
+  }
```

Comment:
I'm not sure if this is quite right -- the definition of "active" for different handle types is fairly involved. What we actually want is just "has this handle every been initialized?". We can obviously track that ourselves, but I think this will also work:

```
if ( UV_UNKNOWN_HANDLE != uv_handle_get_type(&sam_u->nat_u.pol_u) ) {
  uv_close(...)
}
```

`UV_UNKNOWN_HANDLE` is `0`, and we zero-initialize the `uv_poll_t` struct by virtue of allocating the `u3_ames` struct with `c3_calloc()`.